cloud-haven-react-app/src/components/TopBar/TopBar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './TopBar.css';
+import ProfileDropdown from '../ProfileDropdown/ProfileDropdown.js'
+
+function TopBar() {
+   return (
+      <div id="TopBar">
+         <a href="/">
+            <img id="CloudHavenLogo" alt="Cloud Haven" src={require("../../images/CloudHavenLogo.png")}/>
+         </a>
+         <div id="ProfileDropdown" class="float-right">
+            <ProfileDropdown />
+         </div>
```

Comment:
```suggestion
         <ProfileDropdown id="ProfileDropdown"/>
```
You can apply the id to it directly, I'm not sure what the `class="float-right"` is for.

Reply:
`class="float-right"` keeps the button on the right side of the top bar; when I tried using `<ProfileDropdown id="ProfileDropdown" class="float-right"/>` the button wouldn't stay on the right side of the bar.